config/config.go

```diff
@@ -30,8 +30,11 @@ type LockConfig struct {
 }
 
 // ReadTerragruntConfig the Terragrunt config file from its default location
-func ReadTerragruntConfig() (*TerragruntConfig, error) {
-	return parseConfigFile(configFilePath)
+func ReadTerragruntConfig(filePath string) (*TerragruntConfig, error) {
+	if filePath == "" {
+		filePath = configFilePath
+	}
```

Comment:
The CLI portion of the app can be responsible for setting the `filePath` to the default value (`.terragrunt`). If you do that, then you can merge the `parseConfigFile` method and `ReadTerragruntConfig` into one, since they would do the same thing.